packages/host/src/Host.tsx

```diff
@@ -60,45 +60,47 @@ const Host: FC = () => {
           <AppThemeProvider>
             <BrowserRouter>
               <Viewport>
+                <AppBar />
                 <Box display="flex" flex={1}>
-                  <AppBar />
                   <AppDrawer />
-                  <React.Suspense fallback={'Loading'}>
-                    <Routes>
-                      <Route
-                        path="dashboard/*"
-                        element={<DashboardService />}
-                      />
-                      <Route
-                        path="customers/*"
-                        element={<CustomerContainer />}
-                      />
-                      <Route
-                        path="suppliers/*"
-                        element={<Heading locale={locale}>suppliers</Heading>}
-                      />
-                      <Route
-                        path="stock/*"
-                        element={<Heading locale={locale}>stock</Heading>}
-                      />
-                      <Route
-                        path="tools/*"
-                        element={<Heading locale={locale}>tools</Heading>}
-                      />
-                      <Route
-                        path="reports/*"
-                        element={<Heading locale={locale}>reports</Heading>}
-                      />
-                      <Route
-                        path="messages/*"
-                        element={<Heading locale={locale}>messages</Heading>}
-                      />
-                      <Route
-                        path="*"
-                        element={<Navigate to="/dashboard" replace />}
-                      />
-                    </Routes>
-                  </React.Suspense>
+                  <Box flex={1} style={{ marginTop: 90 }}>
```

Comment:
Should this be `72`, the height of the `AppBar`?

Reply:
was looking mainly at [Framework / Table](https://app.zeplin.io/project/60e44fff6fa942168092f1ff/screen/60e498d63f93c6b91456c0b9) which is `90px` - the later screens all seem to be `90` so went with that